paseos/tests/actor_builder_test.py

```diff
@@ -95,3 +95,19 @@ def test_add_comm_device():
     assert len(sat1.communication_devices) == 2
     assert sat1.communication_devices["dev1"].bandwidth_in_kbps == 10
     assert sat1.communication_devices["dev2"].bandwidth_in_kbps == 42
+
+
+def test_set_spacecraft_body_model():
+    """Check if we can set the spacecraft body model, and if the moments of inertia are calculated correctly"""
+    _, sat1, _ = get_default_instance()
+    ActorBuilder.set_spacecraft_body_model(sat1, mass=100)
+
+    assert sat1.mass == 100
+    assert all(
+        sat1.body_center_of_gravity == np.array([0, 0, 0])
+    )  # check the default mesh is centered
+    assert sat1.body_mesh.volume == 1  # check the default volume is correct
+    assert round(sat1.body_moment_of_inertia[0, 0], 4) == 16.6667  # for the default mesh
+    assert (
+        sat1.body_moment_of_inertia[0, 1] == 0.0
+    )  # Should be zero if the mass distribution is even
```

Comment:
as above does not check the computation since it just checks if the default is set